Applications/TransferLearning/Draw_Classification/jni/main.cpp

```diff
@@ -226,97 +243,105 @@ void ExtractFeatures(std::string p, vector<vector<float>> &feature_input,
  * @param[in]  arg 2 : resource path
  */
 int main(int argc, char *argv[]) {
```

Comment:
In the main function, I found that returning negative value logs like #501.
How about return 1 instead?

Reply:
done.